src/Model/Sheet.php

```diff
@@ -35,7 +35,7 @@
  */
 class Sheet extends Table
 {
-    public function initialize(array $config)
+    public function initialize(array $config): void
```

Comment:
This version of the plugin supports CakePHP 3.x and PHP 5.6, so no return types here.